docs/tutorials/sending-and-receiving-messages-with-mqtt.md

```diff
@@ -110,7 +114,10 @@ test it on our PC or on another EV3:
 
 Note: the second EV3 (the "Subscriber") just needs the "paho-mqtt" library,
 there is no need to install the "mosquitto" daemon.
-Note: when the publisher sends a string as payload, convert it in the Subscrider from binary string to ascii as in the example above. When the Publisher sends a number, you can use 'int(mas.payload)' as in the next example.
+
+Note: when the publisher sends a string as payload use 'decode()' as in the
```

Comment:
The example above says `decode('ascii')` rather than just `decode()`. I suggest that the example be changed to just `decode()` as well.